### PR TITLE
ESI-12126 gohan migrate version sometimes fails when run on multiple nodes at once

### PR DESCRIPTION
**Pull request by mmatczuk**

### CODE REVIEW

File: db/migration/migration.go [hunk 1]
```diff
@@ -105,21 +105,51 @@ func Run(subCmd string, args []string) {
...
-		logger.Error("migration: failed to load migration plugins: %s", err)
-		return
+		fmt.Printf("error: failed to load migration plugins: %s\n", err)
+		os.Exit(1)
 	}
 
-	if err != nil {
-		logger.Error("migration: failed to load runtime migrations: %s", err.Error())
-		return
+	if err := lock(dbType, db); err != nil {
```
Comment (p-kozlowski) [inline db/migration/migration.go]: defer unlock. and, by the way: https://github.com/cloudwan/goose/pull/4#discussion_r109954692

Comment (mmatczuk) [inline db/migration/migration.go]: :+1:

**Comment by mmatczuk** [comment]
rebased

**Comment by mmatczuk** [comment]
@marcin-ptaszynski could you take a look at this PR.

File: db/migration/migration.go (hunk 1 above)
Comment (marcin-ptaszynski) [inline db/migration/migration.go]: this review required change hasn't been applied yet.

**Comment by mmatczuk** [comment]
@marcin-ptaszynski done

**Comment by przemyslaw-dobrowolski-cl** [comment]
this PR has been replaced with:
https://github.com/cloudwan/gohan/pull/473

**Pull request closed without merging**